src/components/LikedByList.tsx

```diff
@@ -0,0 +1,137 @@
+import {Loader} from '#/components/Loader'
+import {Text} from '#/components/Typography'
+import {Button, ButtonIcon, ButtonText} from '#/components/Button'
+
+export function LikedByList({uri}: {uri: string}) {
+  const t = useTheme()
+  const {_} = useLingui()
+  const [isPTRing, setIsPTRing] = React.useState(false)
+  const {
+    data: resolvedUri,
+    error: resolveError,
+    isFetching: isFetchingResolvedUri,
+  } = useResolveUriQuery(uri)
+  const {
+    data,
+    isFetching,
+    isFetched,
+    isFetchingNextPage,
+    hasNextPage,
+    fetchNextPage,
+    isError,
+    error,
+    refetch,
+  } = useLikedByQuery(resolvedUri?.uri)
+  const likes = React.useMemo(() => {
+    if (data?.pages) {
+      return data.pages.flatMap(page => page.likes)
+    }
+    return []
+  }, [data])
+
+  const onRefresh = React.useCallback(async () => {
+    setIsPTRing(true)
+    try {
+      await refetch()
+    } catch (err) {
+      logger.error('Failed to refresh likes', {message: err})
+    }
+    setIsPTRing(false)
+  }, [refetch, setIsPTRing])
+
+  const onEndReached = React.useCallback(async () => {
+    if (isFetching || !hasNextPage || isError) return
+    try {
+      await fetchNextPage()
+    } catch (err) {
+      logger.error('Failed to load more likes', {message: err})
+    }
+  }, [isFetching, hasNextPage, isError, fetchNextPage])
+
+  const renderItem = React.useCallback(({item}: {item: GetLikes.Like}) => {
+    return (
+      <ProfileCardWithFollowBtn key={item.actor.did} profile={item.actor} />
+    )
+  }, [])
+
+  if (isFetchingResolvedUri || !isFetched) {
+    return (
+      <View style={[a.w_full, a.align_center, a.p_lg]}>
+        <Loader size="xl" />
+      </View>
+    )
+  }
+
+  if (resolveError || isError) {
+    return (
+      <View style={[a.p_lg]}>
+        <View style={[a.p_lg, a.rounded_sm, t.atoms.bg_contrast_25]}>
+          <Text style={[a.text_md, a.pb_lg]}>
+            {cleanError(resolveError || error)}
+          </Text>
+
+          <View style={[a.flex_row, a.justify_end]}>
+            <Button
+              label={_(msg``)}
+              onPress={onRefresh}
+              size="small"
+              variant="solid"
+              color="primary">
+              <ButtonText>
+                <Trans>Try again</Trans>
+              </ButtonText>
+              <ButtonIcon icon={Refresh} position="right" />
+            </Button>
+          </View>
+        </View>
+      </View>
+    )
+  }
+
+  return likes.length ? (
+    <List
+      data={likes}
+      keyExtractor={item => item.actor.did}
+      refreshing={isPTRing}
+      onRefresh={onRefresh}
+      onEndReached={onEndReached}
+      renderItem={renderItem}
+      initialNumToRender={15}
+      contentContainerStyle={{borderWidth: 0}}
+      // FIXME(dan)
+      // eslint-disable-next-line react/no-unstable-nested-components
+      ListFooterComponent={() => (
+        <View style={[a.w_full, a.align_center, a.p_lg]}>
```

Comment:
We need to add bottom padding to this to prevent the tab bar from overlaying the list. Again though, if we merge in the list fixes before I can just replace this with the reusable footer component. Either works. Alternatively 180px is the right number to add here for bottom padding.

![Screenshot 2024-03-16 at 8 34 10 PM](https://github.com/bluesky-social/social-app/assets/153161762/4ba9c9d9-6fd1-4d15-967d-2bd1a585f424)